clarity_epp/export/caliper.py

```diff
@@ -67,48 +74,40 @@ def samplesheet_normalise(lims, process_id, output_file):
     output_ul = process.udf['Eindvolume (ul) genormaliseerd gDNA']
     output_plate_barcode = process.output_containers()[0].name
 
-    for p in qc_processes:
-        if 'Dx Qubit QC' in p.type.name:
-            for a in p.all_outputs():
-                if 'Tecan' not in a.name and 'check' not in a.name and 'Label' not in a.name:
-                    if 'Dx Conc. goedgekeurde meting (ng/ul)' in a.udf:
-                        machine = 'Qubit'
-                        sample = a.samples[0].name
-                        measurement = a.udf['Dx Conc. goedgekeurde meting (ng/ul)']
-                        if sample in samples_measurements_qubit:
-                            samples_measurements_qubit[sample].append(measurement)
-                        else:
-                            samples_measurements_qubit[sample] = [measurement]
-                    else:
-                        sample = a.samples[0].name
-                        if sample not in sample_concentration:
-                            sample_concentration[sample] = 'geen'
-        elif 'Dx Tecan Spark 10M QC' in p.type.name:
-            for a in p.all_outputs():
-                if 'Tecan' not in a.name and 'check' not in a.name and 'Label' not in a.name:
-                    if 'Dx Conc. goedgekeurde meting (ng/ul)' in a.udf:
-                        machine = 'Tecan'
-                        sample = a.samples[0].name
-                        measurement = a.udf['Dx Conc. goedgekeurde meting (ng/ul)']
-                        if sample in samples_measurements_tecan:
-                            samples_measurements_tecan[sample].append(measurement)
-                        else:
-                            samples_measurements_tecan[sample] = [measurement]
-                    else:
-                        sample = a.samples[0].name
-                        if sample not in sample_concentration:
-                            sample_concentration[sample] = 'geen'
-    for p in qc_processes:
-        for a in p.all_outputs():
-            if 'Tecan' not in a.name and 'check' not in a.name and 'Label' not in a.name:
-                if 'Dx Tecan Spark 10M QC' in p.type.name:
-                    if 'Dx Conc. goedgekeurde meting (ng/ul)' in a.udf:
-                        machine = 'Tecan'
-                    sample = a.samples[0].name
-                elif 'Dx Qubit QC' in p.type.name:
```

Comment:
i never see "machine" referenced in your code

Reply:
See https://github.com/UMCUGenetics/clarity_epp/blob/e8541418f92453225962356d7a2a371bfcf0e2e1/clarity_epp/export/caliper.py#L111